### PR TITLE
Don't perform a query if the delta_magnitude is zero

### PR DESCRIPTION
**Pull request by benjaminwols**
Addresses https://github.com/magnusvk/counter_culture/issues/392#issue-2335855620

### CODE REVIEW

**Comment by benjaminwols** [comment]
@magnusvk I see that half of the sqlite3 tests fail, but I don't think they are related to my change. Any idea on how to fix these failures?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -102,6 +102,22 @@ def db_random
   end
 end
 
+# Spec for checking the number of queries executed within the block
+def expect_queries(num = 1, filter: "", &block)
```
Comment (marinmeister) [inline spec/spec_helper.rb:106]: Nice! I might steal it for https://github.com/magnusvk/counter_culture/pull/393

**Comment by magnusvk** [comment]
@benjaminwols can you rebase on the latest master to pull in the fix for the sqlite test failures?

**Comment by benjaminwols** [comment]
@magnusvk done

**Comment by magnusvk** [comment]
Thank you! I just released this as gem version 3.6.0.

**Pull request merged**